packages/webapi/README.md

```diff
@@ -174,6 +174,4 @@ Code from [fetch-blob](https://www.npmjs.com/package/fetch-blob) is licensed und
 
 Code from [formdata-polyfill](https://www.npmjs.com/package/formdata-polyfill) is licensed under the MIT License (MIT), Copyright Jimmy Wärting.
 
-Code from [node-fetch](https://www.npmjs.com/package/node-fetch) is licensed under the MIT License (MIT), Copyright Node Fetch Team.
-
 Code from [web-streams-polyfill](https://www.npmjs.com/package/web-streams-polyfill) is licensed under the MIT License (MIT), Copyright Mattias Buelens and Diwank Singh Tomer.
```

Comment:
License wasn't removed here

Reply:
It was here.. https://github.com/withastro/astro/pull/5782/commits/10a3e93724d5b04a1225dad19d5a4bcafb6b4705

Reply:
I'm not sure why the Files view is not showing the change...

Reply:
Oh, this is a different file. ok.